### DISCUSSION TITLE
Found 1 proofs,but not chia win

### DISCUSSION THREAD

**Discussion by longren610** [Chia-Network/chia-blockchain#7908] [Q&A] [answered]
On July 31st, there were more than 5,500 "Found 1 proofs" logs, but the wallet did not receive any rewards. Is this normal?

**Comment by longren610** [comment, marked as answer]
I have found an answer, but I don’t know if it’s correct.
https://chiaforum.com/t/please-help-found-proofs-with-no-reward/10002

**Reply by keliew** [reply]
https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#is-it-possible-to-have-a-proof-but-not-get-a-reward

It is, but I'd dig further to see if everything is working fine.

**Comment by emlowe** [comment]
OG plots it is possible - but unlikely as the FAQ suggests
NFT plots with pool - this is expected, as your pool sets the difficulty (and some pools have this set quite low) and you will see many **partial proofs** sent to the pool and the log currently does not differentiate these very well. So you will find many such proofs for the pool, but not farm blocks.
